dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/JarLinkageReport.java

```diff
@@ -28,19 +29,25 @@
   /**
    * Returns the absolute path of the jar file containing source classes of linkage errors
    */
-  public abstract Path getJarPath();
+  abstract Path getJarPath();
+  
+  /**
+   * @return the paths to this artifact
```

Comment:
```suggestion
   * Returns the paths to this artifact.
```

https://google.github.io/styleguide/javaguide.html#s7.2-summary-fragment

Reply:
done